features/controller_specs/headers.feature

```diff
@@ -0,0 +1,50 @@
+Feature: Headers
+
+  We recommend you to switch to request specs instead of controller specs if you want to set
+  headers in your call.
+  If you still want to set headers in controller specs, you can use `request.headers` as
```

Comment:
I'd add a line break to turn this into two paragraphs or run this together into one paragraph.